modules/collection/programs/kitty.nix

```diff
@@ -0,0 +1,91 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: let
+  inherit (lib.attrsets) mapAttrsToList;
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkEnableOption mkOption mkPackageOption;
+  inherit (lib.strings) concatStringsSep optionalString;
+  inherit (lib.types) attrsOf bool float int lines oneOf str;
+
+  cfg = config.rum.programs.kitty;
+in {
+  options.rum.programs.kitty = {
+    enable = mkEnableOption "kitty";
+
+    package = mkPackageOption pkgs "kitty" {};
+
+    settings = mkOption {
+      type = attrsOf (oneOf [float int str]);
```

Comment:
home-manager adds `bool` to this list of types, that converts to `yes` and `no` under-the-hood. Should the same behaviour be replicated here? I'm not sure how much abstraction is too much abstraction